### PR TITLE
example: Nhost with auth and realtime graphql example added

### PR DESCRIPTION
**Pull request by elitan**

### CODE REVIEW

File: examples/with-nhost-auth-realtime-graphql/pages/api/hello.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export default function handler(req, res) {
+  res.statusCode = 200
+  res.json({ name: 'John Doe' })
```
Comment (leerob) [inline examples/with-nhost-auth-realtime-graphql/pages/api/hello.js:3]:
```suggestion
  res.status(200).json({ name: 'John Doe' })
```

**Review by leerob — COMMENTED** [review]
This could likely use just a litttttle bit of styling to make the demo more approachable, but not a blocker.

**Review by leerob — APPROVED** [review]
Excited to see this with an integration! Then we can remove lots of the steps in the README 😄

**Comment by elitan** [comment]
> Excited to see this with an integration! Then we can remove lots of the steps in the README 😄

Yes!

**Pull request merged**